docs/developing_lagoon/index.md

```diff
@@ -30,7 +30,7 @@ If you would still like to build and start all services, go ahead:
 2\. Build images
 
 ```sh
-make build
+make build -j$(getconf _NPROCESSORS_ONLN)
```

Comment:
mmm, I don't think it's safe to just assume we can use all the processors of a system on the build. On our testing Jenkins for instance, this could cause quite the lockup